Jupyter Notebook Deprecation Notice (#1209)
jupyter/docker-stacks: README.md

grant additional permissions (e.g., ability to merge PRs) to anyone who shows an on-going interest
in working on the project.

## Jupyter Notebook Deprecation Notice

Following [Jupyter Notebook notice](https://github.com/jupyter/notebook#notice), we encourage users to transition to JupyterLab.
This can be done by passing the environment variable `JUPYTER_ENABLE_LAB=yes` at container startup, 
more information in the [documentation](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/common.html).
This change is followed in the issue [#1217](https://github.com/jupyter/docker-stacks/issues/1217).

> 

Reviewer: Do we want to state a date after which lab will become the default? April or later maybe?

Author: Yes good idea, April sounds reasonable I think.

grant additional permissions (e.g., ability to merge PRs) to anyone who shows an on-going interest
in working on the project.

## Jupyter Notebook Deprecation Notice

Following [Jupyter Notebook notice](https://github.com/jupyter/notebook#notice), we encourage users to transition to JupyterLab.
This can be done by passing the environment variable `JUPYTER_ENABLE_LAB=yes` at container startup, 

> more information is available in the [documentation](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/common.html).

Reviewer 2: ```suggestion
more information is available in the [documentation](https://jupyter-docker-stacks.readthedocs.io/en/latest/using/common.html#docker-options).
```

I think more specific link is better.

> In April 2021 JupyterLab will become the default, however a new environment variable will be introduced to switch back to Jupyter Notebook if needed.

Reviewer: ```suggestion
In April 2021 JupyterLab will become the default for all of the Jupyter Docker stack images, however a new environment variable will be introduced to switch back to Jupyter Notebook if needed.
```

> In April 2021 JupyterLab will become the default for all of the Jupyter Docker stack images, however a new environment variable will be introduced to switch back to Jupyter Notebook if needed.

Reviewer 3: @parente I'm not sure that April 2021 gives enough time for folks to adjust to this change. It will likely break anyone who hasn't pinned to a SHA.

> Next, and according to the Jupyter Notebook project status and its compatibility with JupyterLab, it could be abandoned in favor of another *classic-like* UI.

Reviewer: ```suggestion
After the change of default, and according to the Jupyter Notebook project status and its compatibility with JupyterLab, these Docker images may remove the classic Jupyter Notebook interface altogether in favor of another *classic-like* UI built atop JupyterLab.
```
